src/index.js

```diff
@@ -118,6 +135,7 @@ class Masq {
           }
         },
         logged: {
+          entry: [ _storeSessionInfo ],
           on: {
             SIGNOUT: {
               target: 'notLogged',
```

Comment:
every target should be prefixed with `#Masq.target` ?

Reply:
#Masq prefix is necessary only when indicating a target located in the 'Masq' machine from another machine.